CMakeLists.txt

```diff
@@ -1,20 +1,8 @@
 
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.2.0)
 
-if(POLICY CMP0017)
-    # Shadow cmake provided modules
-    cmake_policy(SET CMP0017 OLD)
-endif()
-
-if(POLICY CMP0026)
-    # Allow use of the LOCATION property
-    cmake_policy(SET CMP0026 NEW)
```

Comment:
NEW policies are NOT the default, even in newer CMake versions. You need `cmake_policy(VERSION 3.2.0)` for that.

Reply:
The docs read that way but i did test it and it does seem to default to new
https://paste.fedoraproject.org/paste/ap~Zjn6U2~oIuChXG0AQgQ